OutOfSchool/OutOfSchool.WebApi/Services/StatisticService.cs

```diff
@@ -195,10 +195,13 @@ public async Task<IEnumerable<WorkshopCard>> GetPopularWorkshopsFromDatabase(int
     private async Task<List<WorkshopCard>> GetWorkshopsWithAverageRating(List<WorkshopCard> workshopsCards)
     {
         var averageRatings = await averageRatingService.GetByEntityIdsAsync(workshopsCards.Select(p => p.WorkshopId)).ConfigureAwait(false);
+        AverageRatingDto averageRatingDto = null;
```

Comment:
(Code style) This row looks bit excessive. You could just say `var averageRatingDto = averageRatings?....` in `foreach` cycle body and that's it. Don't worry, no performance penalties will be provided.

Reply:
Ok. Done.